air/src/tests.rs

```diff
@@ -0,0 +1,56 @@
+use super::{select_result_range, TransitionConstraintRange};
+use vm_core::utils::range as create_range;
+
+#[test]
+fn transition_constraint_ranges() {
```

Comment:
Why move these tests out of utils? I think they are local to the code there.

If you want to separate tests from the rest of the code a bit better, I'd change utils into a directory with `mod.rs` and `tests.rs` under it.

Reply:
I understood from [this comment](https://github.com/0xPolygonMiden/crypto/pull/67#pullrequestreview-1306060799) that we always leave the tests on a separate module:

>  d. I'd move tests into a separate file (i.e. src/merkle/mmr/tests.rs) - we use this pattern quite frequently.

> If you want to separate tests from the rest of the code a bit better, I'd change utils into a directory with mod.rs and tests.rs under it.

so the idea is to have one `mod.rs` and onde `tests.rs` per folder?

Reply:
I'd say not always - only if a module gets large - but "large" is somewhat arbitrary. Like if a module has 500 lines of non-test code, I'd for sure move the tests into a separate file. I do it sometimes even for smaller modules - especially if there are a lot of tests. For modules under 100 lines I usually don't do it - but also don't mind if it happens.